tests/provision/beaker/data/config/hardware.fmf

```diff
@@ -0,0 +1,6 @@
+beaker:
+  translations:
+    - requirement : cpu.processors
+      template: '{"cpu": {"processors": {"_op": "{{ BEAKER_OPERATOR }}", "_value": "{{ BEAKER_VALUE }}"}}}'
+    - requirement : cpu.model_name
+      template: '{"cpu": {"dummyname": {"_op": "{{ BEAKER_OPERATOR }}", "_value": "{{ BEAKER_VALUE }}"}}}'
```

Comment:
We should make this eaiser to read for us, machines won't care. `template` is supposed to be a YAML, so how about formatting it that way?

```yaml
beaker:
  translations:
    - requirement : cpu.processors
      template: |
        cpu:
          processors:
            _op: "{{ BEAKER_OPERATOR }}"
            _value: "{{ BEAKER_VALUE }}"
```